src/containers/Alerts/alerts.scss

```diff
@@ -4,4 +4,17 @@
 			color: inherit !important;
 		}
 	}
+	.alert-button {
```

Comment:
@Aringoaway Do you really need this styling for button? I am not sure...

Reply:
@Pe5h4 I need this styles, because of i assign button a color and other styles like margin, font-weight ... How do I do it without a class?

Reply:
But why do you need font weight in the button? I thought some is already implemented by default. The same with the color (except the color of the text). You definitely dont need this styling

```
font-weight: bold;
cursor: pointer;
margin-left: 20px;
```

As it was used for the previous version of the implementation (<a> tag or <Link>)